src/applications/ivc-champva/shared/utilities.js

```diff
@@ -77,7 +77,14 @@ export function getConditionalPages(pages, data, index) {
 
 // Expects a date as a string in YYYY-MM-DD format
 export function getAgeInYears(date) {
-  const difference = Date.now() - Date.parse(date);
+  let difference = new Date(Date.now() - Date.parse(date));
+
+  // Get UTC offset to account for local TZ (See https://stackoverflow.com/a/9756226)
+  const utcOffsetSeconds =
+    (difference.getTime() + difference.getTimezoneOffset() * 60 * 1000) / 1000;
+
+  difference -= utcOffsetSeconds;
+
   return Math.abs(new Date(difference).getUTCFullYear() - 1970);
```

Comment:
Updated so that we account for the timezone of where the Date functions are invoked and do all comparisons on UTC dates so that we're comparing apples to apples.